sdk/core/core-rest-pipeline/review/core-rest-pipeline.api.md

```diff
@@ -382,11 +383,16 @@ export interface RetryStrategy {
 export type SendRequest = (request: PipelineRequest) => Promise<PipelineResponse>;
 
 // @public
-export function setClientRequestIdPolicy(requestIdHeaderName?: string): PipelinePolicy;
+export function setClientRequestIdPolicy(options?: SetClientRequestIdPolicyOptions): PipelinePolicy;
```

Comment:
I introduced this breaking because I notice most of policies wrap their options into options bag even they may only have one input. I'd like to follow this rule for better scalability in future.

I reverted this back if we have concern for this breaking.

Reply:
I think in the future if we add a second parameter to the policy, we should absolutely support an options bag. Perhaps we can achieve that by adding an overload to the policy function to avoid the break?

Reply:
Yes, I think overload could help us to avoid any breaking!